AdChoicesManager.js

```diff
@@ -3,22 +3,25 @@ import {requireNativeComponent, StyleSheet, Platform} from 'react-native';
 
 const NativeAdChoicesView = requireNativeComponent('AdChoicesView', null);
 
+type adChoiceLocation = 'topLeft' | 'topRight' | 'bottomLeft' | 'bottomRight';
+
 export default class AdChoicesView extends React.Component<Object> {
-    render() {
+    
+    render() {   
         if (!this.props.placementId) {
             return null;
         }
 
         return (
             <NativeAdChoicesView
                 placementId={this.props.placementId}
-                style={[styles.adChoice, this.getPositionStyle()]}
+                style={[styles.adChoice, this.getPositionStyle(this.props.position)]}
                 location={this.props.position || 'topRight'}/>
```

Comment:
Wouldn't this collide with the default position style? Default location is `topRight` while default position style is `styles.topLeft`

Reply:
Alright i messed it up i should have been sleeping. Already above 27 hrs. Fix it and i'll merge it